src/core/Microsoft.PowerToys.Settings.UI.Lib/ViewModels/ImageResizerViewModel.cs

```diff
@@ -219,58 +224,43 @@ public int Encoder
             }
         }
 
-        public ICommand SaveSizesEventHandler
-        {
-            get
-            {
-                return new RelayCommand(SavesImageSizes);
-            }
-        }
-
-        public ICommand DeleteImageSizeEventHandler
+        public void AddRow()
         {
-            get
+            try
             {
-                return new RelayCommand<int>(DeleteImageSize);
+                ObservableCollection<ImageSize> imageSizes = Sizes;
+                int maxId = imageSizes.Count > 0 ? imageSizes.OrderBy(x => x.Id).Last().Id : -1;
+                ImageSize newSize = new ImageSize(maxId + 1);
+                newSize.PropertyChanged += Size_PropertyChanged;
+                imageSizes.Add(newSize);
+                _advancedSizes = imageSizes;
+                SavesImageSizes(imageSizes);
             }
-        }
-
-        public ICommand AddImageSizeEventHandler
-        {
-            get
+            catch
```

Comment:
What exception are you expecting to catch here?  We probably shouldn't fail silently.

Reply:
no specific exception in mind, the try catch was used while testing, should have been taken out. The code is called we when delete a custom size from image resizer, should I show a pop-up / error message when it fails?

Reply:
I'm confused (not hard to do :)).  If you plan to take out the exception, than we don't need to show a popup, but if you were seeing an exception, then we need to ask ourselves what the best way to handle that exception is.   Sometimes that is a matter of one or more of the following:  fixing a bug; being more defensive around the conditions that lead to the exception; handling and automatically mitigating the issue;  or informing the user of what went wrong, and what they can do to mitigate the issue.

Reply:
I removed the exceptions here: add35f3 since there is not specific failure expected.